.github/workflows/release.yml

```diff
@@ -75,20 +75,23 @@ jobs:
       run: |
         gh release create ${{github.event.inputs.version}} -t ${{github.event.inputs.version}} -n "Changes since last release:"$'\n\n'"$CHANGES"
     - name: Set up QEMU
-      uses: docker/setup-qemu-action@v1
+      uses: docker/setup-qemu-action@v3
+      with:
```

Comment:
not sure this `with` is needed, default should be fine

Reply:
Fixed, thank you! Seems like that default has been changed in the past? Not sure.